runtime/src/lib.rs

```diff
@@ -523,32 +578,50 @@ construct_runtime!(
         Aura: pallet_aura = 23,
         AuraExt: cumulus_pallet_aura_ext = 24,
 
+        // Scheduler
+        Scheduler: pallet_scheduler = 28,
+
+        // Preimage
+        Preimage: pallet_preimage = 29,
+
         // XCM helpers.
         XcmpQueue: cumulus_pallet_xcmp_queue = 30,
         PolkadotXcm: pallet_xcm = 31,
         CumulusXcm: cumulus_pallet_xcm = 32,
         MessageQueue: pallet_message_queue = 33,
 
+        // Contracts
         Contracts: pallet_contracts = 40,
+
+        // Nfts
+        Nfts: pallet_nfts = 50,
```

Comment:
Any reason for not just continuing with 41? Doesn't matter, just interested 🙂

Reply:
Aaah I see AH uses the 50+ range for assets...

Reply:
Not a special opinion from my side. I was mainly leaving some slack just in case.